codegpt.el

```diff
@@ -31,7 +31,7 @@
 
 ;;; Code:
 
-(require 'openai-completion)
+(require 'openai)
```

Comment:
I think we might need both.

```elisp
(require 'openai)
(require 'openai-completion)
```

Reply:
yeah, seems like importing both should work

Reply:
just updated with both @jcs090218